glacier/src/actions/add-field.ts

```diff
@@ -0,0 +1,8 @@
+import {ReduxStandardAction} from "./";
+import {Field} from "../model";
+
+export type AddFieldAction = ReduxStandardAction<"ADD_FIELD", Field[]>;
+
+export function createAddFieldAction(fields: Field[]): AddFieldAction {
+    return {type: "ADD_FIELD", payload: fields};
```

Comment:
The payload also needs to have a reference (uuid) to the data source the field is associated with, yes?